eventio/file_types.py

```diff
@@ -0,0 +1,25 @@
+import gzip
+
+
+def is_gzip(path):
+    '''Test if a file is gzipped by reading its first two bytes and compare
+    to the gzip marker bytes.
+    '''
+    with open(path, 'rb') as f:
+        marker_bytes = f.read(2)
+
+    return marker_bytes[0] == 0x1f and marker_bytes[1] == 0x8b
+
+
+def is_eventio(path):
+    '''
+    Test if a file is an eventio file by checking it's first two bytes
```

Comment:
lying docstring :-D ... `first four bytes` not two

Reply:
yay, copy paste

Reply:
Document intention not implementation :-P